code/__DEFINES/pipe_construction.dm

```diff
@@ -12,6 +12,7 @@ Construction breaks otherwise
 #define PIPE_HE_MANIFOLD		/obj/machinery/atmospherics/pipe/heat_exchanging/manifold
 #define PIPE_HE_4WAYMANIFOLD	/obj/machinery/atmospherics/pipe/heat_exchanging/manifold4w
 #define PIPE_JUNCTION			/obj/machinery/atmospherics/pipe/heat_exchanging/junction
+#define PIPE_LAYER_MANIFOLD		/obj/machinery/atmospherics/pipe/layer_manifold
```

Comment:
Indentation

Reply:
.what about it?
![image](https://user-images.githubusercontent.com/2003111/30891424-259ba9d0-a2e8-11e7-8e52-21d2760047f8.png)